src/FieldEncryptProcessEntities.php

```diff
@@ -173,21 +169,23 @@ protected function process_field(\Drupal\Core\Field\FieldItemListInterface $fiel
       }
     }
 
-    /**
-     * @var $field \Drupal\Core\Field\FieldItemList
-     */
+    /* @var $field \Drupal\Core\Field\FieldItemList */
     $field_value = $field->getValue();
-    foreach($field_value as &$value) {
-      // Process each of the sub fields that exits.
-      $map = $this->fieldEncryptMap[$field_type];
-      foreach($map as $value_name => $service) {
-        if(isset($value[$value_name])){
-          $value[$value_name] = $this->process_value($value[$value_name], $service, $op);
+    $encryption_profile_id = $storage->getThirdPartySetting('field_encrypt', 'encryption_profile', []);
+    $encryption_profile = $this->encryptionProfileManager->getEncryptionProfile($encryption_profile_id);
+
+    // Process the field with the given encryption provider.
+    foreach ($field_value as &$value) {
+      $properties = $storage->getThirdPartySetting('field_encrypt', 'properties', []);
```

Comment:
Does this filter out properties that have been set to encrypt? I am not seeing or understanding how this block of code is doing so.